ga4gh/server/serverconfig.py

```diff
@@ -93,6 +93,20 @@ class GoogleOidcConfig(ProductionConfig):
     OIDC_CLIENT_SECRET = "XXX"
 
 
+class Auth0Config(DevelopmentConfig):
+    """
+    Auth0 configuration
+    """
+    AUTH0_ENABLED = True
```

Comment:
Will you be changing the default to AUTH0_ENABLED = False for the final release?